src/tests/utils/test-utils.tsx

```diff
@@ -6,48 +6,53 @@ import { SnackbarProvider } from 'notistack';
 import { Provider } from 'react-redux';
 import { Router } from 'react-router-dom';
 
-import { store } from '../../common/store';
+import { Store } from '../../common/store';
```

Comment:
I believe that it's better to use `import type` when importing types only, so that we're more guaranteed that it's not kept after transpiling. I believe this is more a problem with babel than with tsc.
see https://www.typescriptlang.org/docs/handbook/modules.html#importing-types for more information